src/components/videos-you-might-like/youtube-video.js

```diff
@@ -3,13 +3,15 @@ import YouTube from "react-youtube";
 const YoutubeVideo = ({ videoId, onReady, className }) => {
   return (
     <YouTube
-      key={videoId}
       videoId={videoId}
       className={className}
-      iframeClassName="relative z-10 w-full h-full rounded-xl shadow-[0px_0px_3px_1px_rgba(255,255,255,1)]"
+      iframeClassName="ytIFrame relative z-10 w-full h-full rounded-xl shadow-[0px_0px_3px_1px_rgba(255,255,255,1)]"
```

Comment:
For consistency: `yt-iframe`:
```suggestion
      iframeClassName="yt-iframe relative z-10 w-full h-full rounded-xl shadow-[0px_0px_3px_1px_rgba(255,255,255,1)]"
```